pydle/features/ircv3/ircv3_1.py

```diff
@@ -26,27 +27,31 @@ async def _rename_user(self, user, new):
 
     ## IRC callbacks.
 
-    async def on_capability_account_notify_available(self, value):
+    @staticmethod
+    async def on_capability_account_notify_available(value):
         """ Take note of user account changes. """
         return True
 
-    async def on_capability_away_notify_available(self, value):
+    @staticmethod
+    async def on_capability_away_notify_available(value):
         """ Take note of AWAY messages. """
         return True
 
-    async def on_capability_extended_join_available(self, value):
+    @staticmethod
+    async def on_capability_extended_join_available(value):
         """ Take note of user account and realname on JOIN. """
         return True
 
-    async def on_capability_multi_prefix_available(self, value):
+    @staticmethod
+    async def on_capability_multi_prefix_available(value):
         """ Thanks to how underlying client code works we already support multiple prefixes. """
         return True
 
-    async def on_capability_tls_available(self, value):
+    @staticmethod
+    async def on_capability_tls_available(value):
```

Comment:
Please revert these changes. I do not wish to impact pydle's API with this changeset.